pkgs/tools/misc/hyfetch/default.nix

```
@@ -0,0 +1,36 @@
+{ python3Packages, lib, fetchFromGitHub }:
```

The policy is `lib` first, then sort by usage order.
```suggestion
{ lib, python3Packages, fetchFromGitHub }:
```